individual-code-of-conduct.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: page
+menu_id: download
+title: Code of Conduct
+---
+
+# Presto Code of Conduct
+
+Like the technical community as a whole, the Presto team and community is made up of a mixture of professionals and
+volunteers from all over the world, working on every aspect of Presto.
+
+Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we 
+have a few ground rules that we ask people to adhere to. This code applies equally to all participants in the Presto
+project.
+
+This isn’t an exhaustive list of things that you can’t do. Rather, take it in the spirit in which it’s intended - a 
+guide to make it easier to enrich all of us and the technical communities in which we participate.
+
+This code of conduct applies to all spaces managed by the Presto project or Presto Software Foundation. This includes 
+Slack, the mailing lists, the issue tracker, PSF events, and any other forums created by the project team which the
+community uses for communication. In addition, violations of this code outside these spaces may affect a person's
+ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you report it by emailing `conduct@prestosql.io`. 
+-- For more details please see our Reporting Guidelines
+
+* **Be friendly and patient.**
+* **Be welcoming**. We strive to be a community that welcomes and supports people of all backgrounds and identities.
+This includes, but is not limited to members of any race, ethnicity, culture, national origin, colour, immigration
+status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, 
+size, family status, political belief, religion, and mental and physical ability.
+* **Be considerate.** Your work will be used by other people, and you in turn will depend on the work of others. Any
+decision you take will affect users and colleagues, and you should take those consequences into account when making 
+decisions. Remember that we're a world-wide community, so you might not be communicating in someone else's primary 
+language.
+* **Be respectful.** Not all of us will agree all the time, but disagreement is no excuse for poor behavior and poor
+manners. We might all experience some frustration now and then, but we cannot allow that frustration to turn into a
+personal attack. It’s important to remember that a community where people feel uncomfortable or threatened is not a 
+productive one. Members of the Presto community should be respectful when dealing with other members as well as with 
+people outside the Presto community.
+* **Be careful in the words that you choose.** We are a community of professionals, and we conduct ourselves
+professionally. Be kind to others. Do not insult or put down other participants. Harassment and other exclusionary 
```

Comment:
> put down

can you think of expressing this without using an idiom?
Just to make it simpler to understand or even translate.

Reply:
I change it to `disparage`